src/Auth0Client.ts

```diff
@@ -115,6 +115,9 @@ const getTokenIssuer = (issuer: string, domainUrl: string) => {
   return `${domainUrl}/`;
 };
 
+/**
+ * @ignore
+ */
```

Comment:
I ignored this function as it is an internal helper and we dont want it to be documented.